What is the file_header rule about?

I do not understand what is the issue reported by the rule `file_header`.

```
//
//  ActivityModel.swift
//  English 24/7
//
//  Created by Antoine Cœur on 8/24/16.
//  Copyright © 2016 EF Education. All rights reserved.
//
```
> /MyApp/ActivityModel.swift:6:5: warning: File Header Violation: Files should have consistent header comments. (file_header)

What does it mean?
Does it mean I need to change the Copyright year for all my files on every January 1st?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -74,6 +74,10 @@
   [Samuel Susla](https://github.com/sammy-SC)
   [#1387](https://github.com/realm/SwiftLint/issues/1165)
 
+* Improve `file_header` rule description.  
+  [Marcelo Fabri](https://github.com/marcelofabri)
+  [#1492](https://github.com/realm/SwiftLint/issues/1492)
+
 ##### Bug Fixes
 
 * Fix false positive on `redundant_discardable_let` rule when using

--- a/Rules.md
+++ b/Rules.md
@@ -2302,7 +2302,7 @@ Identifier | Enabled by default | Supports autocorrection | Kind
 --- | --- | --- | ---
 `file_header` | Disabled | No | style
 
-Files should have consistent header comments.
+Header comments should be consistent with project patterns.
 
 ### Examples
 

--- a/Source/SwiftLintFramework/Rules/FileHeaderRule.swift
+++ b/Source/SwiftLintFramework/Rules/FileHeaderRule.swift
@@ -17,7 +17,7 @@ public struct FileHeaderRule: ConfigurationProviderRule, OptInRule {
     public static let description = RuleDescription(
         identifier: "file_header",
         name: "File Header",
-        description: "Files should have consistent header comments.",
+        description: "Header comments should be consistent with project patterns.",
         kind: .style,
         nonTriggeringExamples: [
             "let foo = \"Copyright\"",

